lib/ansible/modules/cloud/vmware/vmware_vm_shell.py

```diff
@@ -63,6 +63,12 @@
       - The VMware identification method by which the virtual machine will be identified.
       default: vm_name
       choices: ['uuid', 'dns_name', 'inventory_path', 'vm_name']
+    vm_uuid_type:
```

Comment:
Makes more sense to use the vm_id_type field, and just add the option `instance_uuid`. Also adjust code, of course